frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnFieldHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+export interface MultiColumnFieldHeaderProps {
+  headers: string[];
+}
+
+const MultiColumnFieldHeader: React.FC<MultiColumnFieldHeaderProps> = ({ headers }) => (
+  <div className="odc-multi-column-field__row">
```

Comment:
`className` belongs in its own file imported by this component file and following the naming conventions to align with the component name.